client/payment-methods/index.js

```diff
@@ -122,9 +122,10 @@ const PaymentMethods = () => {
 
 	const availablePaymentMethodIds = useGetAvailablePaymentMethodIds();
 
-	const availableMethods = availablePaymentMethodIds.map(
-		( methodId ) => methodsConfiguration[ methodId ]
-	);
+	//we filter link payment method since this will be displayed in other section ( express checkout )
+	const availableMethods = availablePaymentMethodIds
+		.filter( ( id ) => 'link' !== id )
+		.map( ( methodId ) => methodsConfiguration[ methodId ] );
```

Comment:
```suggestion
		.filter( id => 'link' !== id )
		.map( methodId => methodsConfiguration[ methodId ] );
```

Minor: Unnecessary `()`